tests/scripts/task_python_vta_fsim.sh

```diff
@@ -20,6 +20,10 @@ set -e
 set -u
 
 source tests/scripts/setup-pytest-env.sh
+# to avoid CI thread throttling.
+export TVM_BIND_THREADS=0
+export OMP_NUM_THREADS=1
+
```

Comment:
Why not move this into setup-pytest-env.sh and maybe rename it to setup-test-env.sh if you think it's too pytest centric ?

Reply:
Thanks for the reply. Agreed on the rust and cpp environments which is why I suggested setup-test-env.sh :). On the second point I would have thought the lower level driver script be able to override the environment variable for the test ? 

I'd like to understand more about what you mean by directly limiting the resources of the container. 

regards
Ramana

Reply:
I meant https://docs.docker.com/config/containers/resource_constraints/, to directly restrict the CPUs that a container can access physically. although we might need some experimentation to see which option is the best, and how are they presented to the runtime.